cdk/lib/clamav-scanner-stack.ts

```diff
@@ -39,6 +38,8 @@ export class ClamavScannerStack extends Stack {
       }),
     });
 
+    props.clamavFileSystem.grantRootAccess(clamavTaskDef.taskRole.grantPrincipal);
```

Comment:
```suggestion
    props.clamavFileSystem.grantRootAccess(clamavTaskDef.taskRole);
```

Propably doesn't need to be given grant principal explicitly.